staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -422,7 +422,7 @@ func (s *GenericAPIServer) installAPIResources(apiPrefix string, apiGroupInfo *A
 	var resourceInfos []*storageversion.ResourceInfo
 	for _, groupVersion := range apiGroupInfo.PrioritizedVersions {
 		if len(apiGroupInfo.VersionedResourcesStorageMap[groupVersion.Version]) == 0 {
-			klog.Warningf("Skipping API %v because it has no resources.", groupVersion)
+			klog.InfoS("Skipping API because it has no resources.", "group", groupVersion.Group, "version", groupVersion.Version)
```

Comment:
does changing from warning to info lower the visibility of what is likely an accident?

Reply:
`klog` does not provide a `WarningS()` method, according to the migration guide, `klog.Warningf` can be migrate to `klog.InfoS`.
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#change-log-functions-to-structured-equivalent

Reply:
> does changing from warning to info lower the visibility of what is likely an accident?

@liggitt Sorry about that, but `klog` does not provide a `WarningS()` method, according to the migration guide,` klog.Warningf` should be migrate to `klog.InfoS`.
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#change-log-functions-to-structured-equivalent